db-server/test/backend/db_tests.js

```diff
@@ -2147,6 +2154,92 @@ module.exports = function (config, DB) {
       })
     })
 
+    describe('account recovery key', () => {
+      let account, data
+      beforeEach(() => {
+        account = createAccount()
+        account.emailVerified = true
```

Comment:
Why is this `emailVerified` part necessary, do we have some logic that prevents creating recovery data on unverified accounts?

Reply:
Nope, no dependency, removed it as well.